.github/mergify.yml

```diff
@@ -12,9 +12,9 @@ pull_request_rules:
   - name: Prepare for merge
     conditions:
       - and:
-        - "-draft"
-        - "#approved-reviews-by=1"
-        - "#review-requested=0"
+          - "-draft"
+          - "#approved-reviews-by=1"
+          - "#review-requested=0"
```

Comment:
Seems like formatting error to me. Shouldn't all `- ...` be indented more here?

Reply:
No? But I think it comes from prettier's formatting

Reply:
Ok then :+1: